client/src/utils.ts

```diff
@@ -31,23 +31,31 @@ export const Token = {
 export async function isReadyForConnect(
   appId: string,
   roomId: string,
-  hathoraSdk: HathoraCloud
+  hathoraSdk: HathoraCloud,
 ): Promise<{ lobbyInfo: LobbyV3; connectionInfo: ConnectionDetails }> {
   const MAX_CONNECT_ATTEMPTS = 50;
   const TRY_CONNECT_INTERVAL_MS = 1000;
 
   try {
-    const lobbyInfo = await hathoraSdk.lobbyV3.getLobbyInfoByRoomId(roomId);
+    const lobbyInfo = await hathoraSdk.lobbiesV3.getLobbyInfoByRoomId(roomId);
     if (lobbyInfo.visibility === "local") {
       return new Promise<{ lobbyInfo: LobbyV3; connectionInfo: ConnectionDetails }>((resolve) =>
-        resolve({ lobbyInfo, connectionInfo: LOCAL_CONNECTION_DETAILS })
+        resolve({ lobbyInfo, connectionInfo: LOCAL_CONNECTION_DETAILS }),
```

Comment:
hmm are these extra commas from ESLint or something? they look wrong to me though.. is this a new standard or something? (I don't like change 👴🏻👴🏻👴🏻)

Reply:
lol yea i think these are eslint--idk what changed--you wanna send me your eslintrc or something?

Reply:
I don't think we had a linter setup when we did the bulk of the dev.

Do we require trailing commas for functions in Cirrus/Console? Fine with linting just being consistent with console/cirrus (even if it means trailing function commas)

I found this eslint thread: https://github.com/eslint/eslint/issues/7851#issuecomment-270428874